src/gotcha.c

```diff
@@ -47,6 +47,48 @@ static void setInternalBindingAddressPointer(void** in, void* value){
   writeAddress(target, value);
 }
 
+long lookup_exported_symbol(const char* name, const struct link_map *lib, void** symbol) {
+    long result;
+    if (is_vdso(lib)) {
+        debug_printf(2, "Skipping VDSO library at 0x%lx with name %s\n",
+                     lib->l_addr, LIB_NAME(lib));
+        return -1;
+    }
+    debug_printf(2, "Searching for exported symbols in %s\n", LIB_NAME(lib));
+    INIT_DYNAMIC(lib);
+
+    if (!gnu_hash && !elf_hash) {
+        debug_printf(3, "Library %s does not export or import symbols\n", LIB_NAME(lib));
+        return -1;
+    }
+    result = -1;
+    if (gnu_hash) {
+        debug_printf(3, "Checking GNU hash for %s in %s\n", name);
```

Comment:
Also, this printf is missing a parameter

Reply:
fixed.